### PR TITLE
bump nix

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Partly because the current version is from February (of this year, fortunately), and partly because I need a more recent one to bump node to 18 and fix the GSG for that.

### CODE REVIEW

**Review by remyhaemmerle-da — APPROVED** [review]

File: compiler/repl-service/client/src/DA/Daml/LF/ReplClient.hs [hunk 1]
```diff
@@ -25,6 +25,7 @@ module DA.Daml.LF.ReplClient
 import Control.Concurrent.Async
 import Control.Concurrent.Extra
 import Control.Exception
+import qualified Control.Monad (void)
```
Comment (S11001001) [inline compiler/repl-service/client/src/DA/Daml/LF/ReplClient.hs:28]: I'd pull this from `Data.Functor` instead.

**Review by S11001001 — APPROVED** [review]

File: compiler/repl-service/client/src/DA/Daml/LF/ReplClient.hs (hunk 1 above)
Comment (garyverhaegen-da) [reply compiler/repl-service/client/src/DA/Daml/LF/ReplClient.hs]: Done.

**Comment by stefanobaghino-da** [comment]
Any specific action to take on individual laptops that needs to happen when pulling after this is merged to `main`?

**Comment by garyverhaegen-da** [comment]
> Any specific action to take on individual laptops that needs to happen when pulling after this is merged to `main`?

`cd` out and back in should be all that you need to do. I'll announce it on #team-daml when this gets merged, but I still need a product person to approve the PostgreSQL change first.

**Comment by stefanobaghino-da** [comment]
> > Any specific action to take on individual laptops that needs to happen when pulling after this is merged to `main`?
> 
> `cd` out and back in should be all that you need to do. I'll announce it on #team-daml when this gets merged, but I still need a product person to approve the PostgreSQL change first.

To the best of my understanding we still support PostgreSQL 10 ([source](https://docs.daml.com/ops/requirements.html) as of 2.4.0 -- and I'm not aware of changes from 2.5.0). If we need to move on to PostgreSQL 11 for some integration testing matrix, we should still have PostgreSQL 10 in since we still support it.

**Comment by garyverhaegen-da** [comment]
> > > Any specific action to take on individual laptops that needs to happen when pulling after this is merged to `main`?
> > 
> > 
> > `cd` out and back in should be all that you need to do. I'll announce it on #team-daml when this gets merged, but I still need a product person to approve the PostgreSQL change first.
> 
> To the best of my understanding we still support PostgreSQL 10 ([source](https://docs.daml.com/ops/requirements.html) as of 2.4.0 -- and I'm not aware of changes from 2.5.0). If we need to move on to PostgreSQL 11 for some integration testing matrix, we should still have PostgreSQL 10 in since we still support it.

PostgreSQL does not support PostgreSQL 10 anymore and therefore it is not part of this nixpkgs snapshot. There is no way to upgrade to this snapshot without bumping our test harness to use 11 instead of 10 (which this PR does, as well as updating the docs page you reference). I'm not asking whether we still support 10 with 2.4.0 (I know we do), I'm asking if we can drop it for 2.5.0.

**Comment by stefanobaghino-da** [comment]
Green light from @bame-da to move on with PostgreSQL 11. I'll take care of documenting this.

**Comment by stefanobaghino-da** [comment]
Documented as part of https://github.com/digital-asset/daml/pull/15670

**Comment by garyverhaegen-da** [comment]
> Green light from @bame-da to move on with PostgreSQL 11. I'll take care of documenting this.

Thanks!

**Pull request merged**